### DISCUSSION TITLE
Is it possible to set the notifyUsers=false

### DISCUSSION THREAD

**Discussion by Noloxs** [ankitpokhrel/jira-cli#591] [Q&A]
Hi.

Is it possible to set notifyUsers=false so that when I perform an edit watchers aren't notified.

I am preparing a script to update every single Jira issue, so it would be much appreciated if it would be possible.

**Comment by opctim** [comment]
@ankitpokhrel bump

**Comment by ankitpokhrel** [comment]
This is done with #855 and will be available in the next release.
